### ISSUE TITLE
Fails packaging on Python 3.8 - No such file or directory ...

### ISSUE THREAD

**Issue by tmontes** [mu-editor/pup#37]
FACT:

* After having fixed #35, `pup` is now failing on macOS with Python 3.8.
* Here's a sample output, trying to package `mu`:
```
$ PUP_LOG_LEVEL=info pup package --launch-module=mu ../../Python/github.com/mu
20200916 125819 I pup 1.0.0a1 - starting with PID=36760
20200916 125819 I pup.api Package '../../Python/github.com/mu': starting.
20200916 125819 I pup.plugins.metadata Collecting metadata for '../../Python/github.com/mu'.
20200916 125822 I pup.api Step 'mac.app-bundle-template': starting.
20200916 125822 I pup.api Step 'mac.app-bundle-template': completed.
20200916 125822 I pup.api Step 'pup.python-runtime': starting.
20200916 125824 E pup Execution failure: [Errno 2] No such file or directory: '/Users/tiago.montes/Work/Python-Public/x/build/pup/mu-editor.app/Contents/Resources/Python/lib/python3.7/config-3.7m-darwin'
20200916 125824 I pup 1.0.0a1 - done
```

MOTIVE:

* The quick-and-very-very-dirty "Python Runtime cleanup" is hard-coded for 3.7. :-)

PROPOSED FIX:

* Let's drop the runtime cleanup hacks and create an issue to handle that (kind of) "properly".

**Comment by tmontes** [comment]
Created #38 to properly follow up on the underlying issue.

**Comment by tmontes** [comment]
Closing after having merged PR #39.
